servers/rendering/renderer_rd/storage_rd/texture_storage.cpp

```diff
@@ -844,7 +844,7 @@ void TextureStorage::texture_2d_layered_initialize(RID p_texture, const Vector<R
 		}
 	}
 
-	Texture texture;
+	Texture texture = {};
```

Comment:
Texture does not have default initializers, which is bad form, but this should be a separate PR.

I don't think we should be adding a single = {} just in this one place since it doesn't improve the overall code quality.

If there is a bug that we're working around or fixing by doing this, I'd like to be explicit about what it is. I double checked all the class members without default initializers and they should all be getting initialized here, so I'm skeptical that this change is needed.